azext_edge/edge/providers/support/base.py

```diff
@@ -174,6 +174,7 @@ def process_deployments(
 
 def process_statefulset(
     directory_path: str,
+    return_namespaces: bool = False,
```

Comment:
btw if this is mq only, helps to keep it in mq (unless there is potential for other services to do this too)

Reply:
talked offline